docs/pages/preview/custom-build-config-schema.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: EAS Build custom builds config schema
+description: A reference of configuration options for custom workflows with EAS Build.
+hideFromSearch: true
+---
+
+Creating custom workflows for EAS Build helps customize the build process by adding your workflows.
+
+## YAML syntax for workflows
+
+Workflow files are stored inside the **.eas/build** directory path. They use YAML syntax and must have a `.yml` or `.yaml` file extension. If you are new to YAML or want to learn more about the syntax, see [Learn YAML in Y minutes](https://learnxinyminutes.com/docs/yaml/).
+
+## `build`
+
+Defined to describe a custom workflow. All config options to create a workflow are specified under it.
+
+### `name`
+
+The name of your workflow that is used to identify the workflow in the build logs. EAS Build uses this property to display the name of your workflow in the dashboard.
+
+For example, the workflow's name is "Run tests":
+
+```yaml
+build:
+  # @info #
+  name: Run tests
+  # @end #
+  steps:
+    - run:
+        name: Install dependencies
+        command: npm install
+```
+
+### `steps`
+
+Steps are used to describe a list of actions either in the form of commands or functions. These actions are executed when a workflow is triggered. You can define single or multiple steps in a workflow. It is **required** to define at least one step per workflow.
+
+Each step is configured with the following properties.
+
+#### `steps.run`
+
+The `run` key is used to trigger a set of instructions. For example, a `run` key is used to install dependencies using the `npm install` command:
```

Comment:
Yeah, I don't think the example below or the definition above for `run` signifies that. I think, since you are pointing this out, it might be important for the specific use case to showcase that there is a possibility for the end-developer to do so.

Do you think updating the example below as you suggested will be a good option here? Because, in `steps.command` we are already specifying that the executing multiline bash scripts is possible. Initially, I didn't want to include this specific phrase in the `run` because I think running multiline bash scripts isn't possible without using `command` key (correct me if my assumption is wrong here).